src/index.js

```diff
@@ -92,11 +92,11 @@ const getUnvisit = (entities) => {
       return method(schema, input, unvisit);
     }
 
-    if (input === undefined || input === null) {
-      return [input, true];
-    }
-
     if (schema instanceof EntitySchema) {
+      if (input === undefined || input === null) {
```

Comment:
this only short-circuits entities as it needs to compute 'found' properly on all others so it should pass-through